src/main/java/org/killbill/billing/plugin/adyen/api/mapping/UserDataMappingService.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2014-2015 Groupon, Inc
+ * Copyright 2014-2015 The Billing Project, LLC
+ *
+ * The Billing Project licenses this file to you under the Apache License, version 2.0
+ * (the "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at:
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package org.killbill.billing.plugin.adyen.api.mapping;
+
+import java.util.Locale;
+
+import javax.annotation.Nullable;
+
+import org.apache.commons.lang3.LocaleUtils;
```

Comment:
Yeah, let's keep the current behavior, and open a new PR to patch and verify the locale fix.

Thanks!